### PR TITLE
Fix Docker button, add PikaPods as automated install option

### PR DESCRIPTION
**Pull request by m3nu**
This fixes a broken Docker image (previously [here](https://www.docker.com/sites/default/files/horizontal.png)) in the README and adds [PikaPods.com](https://www.pikapods.com/) as additional automated install option.

About PikaPods: Our mission is to make great open source apps, like FreshRSS more accessible and easier to get started with. We already offer FreshRSS in our “app store” and we fine-tuned our deployment based on the feedback we got. Currently we have 30 uses running FreshRSS and about 1500 active users (who ran at least one pod).

So I'm suggesting to add PikaPods as additional install option to run FreshRSS. This would help an even wider audience to benefit from the project.

Rendered [preview](https://github.com/m3nu/FreshRSS/tree/misc/add-pikapods-deploy#automated-install)

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -65,9 +65,10 @@ Si vous voulez une publication continue (rolling release) avec les dernières no
 
 ## Installation automatisée
 
-* [![Docker](https://www.docker.com/sites/default/files/horizontal.png)](./Docker/)
+* [![Docker](./docs/img/docker-horizontal.png)](./Docker/)
```
Comment (Alkarex) [inline README.fr.md]: Thanks for the fix. I think I would rather like to keep hot-linking to an image from [Docker.com](https://www.docker.com/company/newsroom/media-resources/) if possible, to ease any logo rights issue

Comment (Alkarex) [inline README.fr.md]:
```suggestion
* [<img src="https://www.docker.com/wp-content/uploads/2022/03/horizontal-logo-monochromatic-white.png" width="200" alt="Docker" />](./Docker/)
```

File: README.md [hunk 2]
```diff
@@ -65,9 +65,10 @@ If you want a rolling release with the newest features, or want to help testing
 
 ## Automated install
 
-* [![Docker](https://www.docker.com/sites/default/files/horizontal.png)](./Docker/)
+* [![Docker](./docs/img/docker-horizontal.png)](./Docker/)
```
Comment (Alkarex) [inline README.md]:
```suggestion
* [<img src="https://www.docker.com/wp-content/uploads/2022/03/horizontal-logo-monochromatic-white.png" width="200" alt="Docker" />](./Docker/)
```

**Comment by Alkarex** [comment]
Please double-check my Docker logo fix. I cannot find a pure Markdown syntax to resize an image, hence the HTML code. Alternative suggestions welcome

File: README.fr.md (hunk 1 above)
Comment (m3nu) [reply README.fr.md]: Both fine with me. I removed the image from the PR as well.

**Comment by m3nu** [comment]
I also don't know a way to scale the image and usually use HTML. 😬

**Pull request merged**